### PR TITLE
Create pwa-features-2020-comparison.md

### PR DESCRIPTION
**Pull request by arthurp1**
I've added 2 different Markdown styles because I don't know which formatting will look well.

### CODE REVIEW

**Comment by marcoow** [comment]
@arthurp1 this requires some fixes but I can take over, don't worry about understanding all of the (rather complex) stuff here…

**Comment by marcoow** [comment]
@arthurp1 I fixed the markdown, reviewed the text etc. – please check again. I think we should publish tomorrow.

@ghislaineguerin can you add an og-image please?

**Comment by marcoow** [comment]
@arthurp1 please also give me your author image

**Comment by arthurp1** [comment]
[author_image_AP](https://media-exp1.licdn.com/dms/image/C4E03AQGPk_OhWwSqig/profile-displayphoto-shrink_400_400/0?e=1596672000&v=beta&t=cWlVq5ViTd2iyPd2BGj6ZnWOSqj_DrSprUxVJ1rABd8)

File: _posts/2020-06-05-pwa-features-2020-comparison.md [hunk 1]
```diff
@@ -65,16 +65,16 @@ other hand, Apple has been very protective of PWAs accessing hardware or app
...
 many native sensors in a desktop device, except hardware that plugs directly
-into the OS environment (e.g. bluetooth headphones, harddrives, or USB devices).
+into the OS environment (e.g. bluetooth headphones, hard drives, or USB devices).
 
 ### What native software features are accessible by PWAs?
 
 For a PWA to be fully indistinguishable from a native app, it needs to blend in
-with the operating softwares native user experience. For example, native push
-notifications, keyboard shortcuts and gestures give that native feeling.
+with the native user interactions offered by the operating software. For example, native push
```
Comment (marcoow) [inline _posts/2020-06-05-pwa-features-2020-comparison.md]:
```suggestion
with the native user interactions offered by the operating system. For example, native push
```

File: _posts/2020-06-05-pwa-features-2020-comparison.md [hunk 2]
```diff
@@ -93,14 +93,14 @@ start comparing it is good to answer the following questions:
 
 Each OS has different users. Designers, developers and the youth use macOS,
 while employees of many corporates spend most of their time on Windows PCs. iOS
-is not as big as Android. Only in the US more than 20% of the online population
-are on iPhones or iPads. So it really depends on what audience you try to serve.
+is not as big as Android. Only in the United States, more than 20% of the online population
+is on an iPhone or iPad. So it really depends on what audience you try to serve.
```
Comment (marcoow) [inline _posts/2020-06-05-pwa-features-2020-comparison.md]:
```suggestion
are on iPhones or iPads. So it really depends on what audience you try to serve.
```

To me that sounds better than referring to singular devices.

**Review by marcoow — CHANGES_REQUESTED** [review]

**Comment by arthurp1** [comment]
I resolved the requested changes. I see the percy/simplabs.com checks are unsuccessful. Is this important, could you check it? Anything else before we could post it? @marcoow

File: _posts/2020-06-05-pwa-features-2020-comparison.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: 'PWA support on mobile and desktop in 2020'
```
Comment (marcoow) [inline _posts/2020-06-05-pwa-features-2020-comparison.md]:
```suggestion
title: 'The state of PWA support on mobile and desktop in 2020'
```

**Review by marcoow — COMMENTED** [review]
@arthurp1 I'd suggest a more expressive title?

**Comment by arthurp1** [comment]
Can I merge it?

**Comment by marcoow** [comment]
@arthurp1 no, the `og:image` is missing – see https://app.abstract.com/projects/c2d6fda0-836d-4507-92b5-635b3e333b4d/branches/3ad212eb-81a6-44ec-98fa-25b40c5f3c09

**Review by marcoow — APPROVED** [review]

**Comment by arthurp1** [comment]
> @arthurp1 no, the `og:image` is missing – see https://app.abstract.com/projects/c2d6fda0-836d-4507-92b5-635b3e333b4d/branches/3ad212eb-81a6-44ec-98fa-25b40c5f3c09

@ghislaineguerin can you help with this?

**Pull request merged**